static/js/src/ua-payment-modal.js

```diff
@@ -147,11 +151,15 @@ function attachCTAevents() {
       currentTransaction.previousPurchaseId = data.previousPurchaseId;
       cartItems.forEach((item) => {
         currentTransaction.products.push({
+          name: item.product.name,
+          price: item.product.price.value,
           product_listing_id: item.listingID,
           quantity: parseInt(item.quantity),
         });
       });
 
+      checkoutEvent(analyticsFriendlyProducts(), "Proceeded to checkout");
```

Comment:
Steps should be integers.

```suggestion
      checkoutEvent(analyticsFriendlyProducts(), 1);
```

Reply:
Ahhhh - I had misunderstood this to mean we could pass strings: "Optionally set user-friendly step names for the checkout funnel report by configuring Ecommerce Settings in the Admin section of the web interface."